server/src/main/java/org/opensearch/search/DefaultSearchContext.java

```diff
@@ -890,11 +890,15 @@ public boolean shouldUseConcurrentSearch() {
      * Evaluate if parsed request supports concurrent segment search
      */
     public void evaluateRequestShouldUseConcurrentSearch() {
-        if (aggregations() != null && aggregations().factories() != null) {
-            requestShouldUseConcurrentSearch.set(aggregations().factories().allFactoriesSupportConcurrentSearch());
-        } else {
-            requestShouldUseConcurrentSearch.set(true);
-        }
+        if (sort != null && sort.isSortOnTimeSeriesField()) {
+            requestShouldUseConcurrentSearch.set(false);
+        } else if (aggregations() != null
+            && aggregations().factories() != null
+            && !aggregations().factories().allFactoriesSupportConcurrentSearch()) {
+                requestShouldUseConcurrentSearch.set(false);
+            } else {
+                requestShouldUseConcurrentSearch.set(true);
+            }
```

Comment:
Looks like there's an extra tab here?

Reply:
Spotless is adding it. Not sure how to fix this.